app/graphql/types/query_type.rb

```diff
@@ -395,4 +395,36 @@ def franchises
   #   DESCRIPTION
   #   argument :title, String, required: true
   # end
+
+  field :reports, Types::Report.connection_type, null: true do
+    description 'All Reports in the Kitsu database'
+  end
+
+  def reports
+    ReportPolicy::Scope.new(context[:token], ::Report).resolve
```

Comment:
You probably want to add a sort to this, even if it's only `.order(created_at: :desc)` 😅